package.json

```diff
@@ -1,56 +1,67 @@
 {
   "name": "powerbi-visuals-chordchart",
-  "version": "1.5.0",
+  "version": "2.0.0",
   "author": {
     "name": "Microsoft",
     "email": "pbicvsupport@microsoft.com"
   },
   "scripts": {
-    "postinstall": "pbiviz update 2.1.0",
     "pbiviz": "pbiviz",
     "start": "pbiviz start",
     "package": "pbiviz package",
-    "cert": "pbiviz --create-cert",
     "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src|test)/**/*.ts\"",
     "pretest": "pbiviz package --resources --no-minify --no-pbiviz --no-plugin",
-    "test": "karma start"
+    "test": "karma start",
+    "cert": "pbiviz --install-cert"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/powerbi-visuals-chord.git"
   },
   "devDependencies": {
-    "@types/d3": "3.5.36",
+    "@types/d3": "5.0.0",
     "@types/jasmine": "2.5.37",
     "@types/jasmine-jquery": "1.5.28",
     "@types/jquery": "2.0.46",
-    "@types/lodash": "4.14.52",
+    "@types/lodash": "4.14.109",
+    "@types/webpack": "4.4.4",
     "coveralls": "2.11.15",
+    "css-loader": "1.0.0",
+    "d3": "^5.7.0",
     "jasmine": "2.5.2",
     "jasmine-jquery": "2.1.1",
+    "jquery": "3.2.1",
     "karma": "2.0.4",
     "karma-chrome-launcher": "2.2.0",
     "karma-coverage": "1.1.2",
     "karma-jasmine": "1.1.2",
     "karma-remap-istanbul": "0.6.0",
     "karma-sourcemap-loader": "0.3.7",
-    "karma-typescript-preprocessor": "0.3.0",
+    "karma-typescript": "3.0.12",
+    "karma-typescript-preprocessor": "0.3.1",
+    "karma-webpack": "3.0.0",
+    "less": "2.1.1",
+    "less-loader": "4.1.0",
     "lodash": "4.16.2",
-    "powerbi-models": "1.0.3",
-    "powerbi-visuals-tools": "2.1.0",
-    "powerbi-visuals-utils-chartutils": "1.5.1",
-    "powerbi-visuals-utils-colorutils": "1.1.0",
-    "powerbi-visuals-utils-dataviewutils": "1.4.1",
-    "powerbi-visuals-utils-formattingutils": "3.0.1",
-    "powerbi-visuals-utils-interactivityutils": "3.1.0",
-    "powerbi-visuals-utils-svgutils": "1.1.0",
-    "powerbi-visuals-utils-testutils": "1.2.0",
-    "powerbi-visuals-utils-tooltiputils": "1.0.0",
-    "powerbi-visuals-utils-typeutils": "1.1.0",
-    "puppeteer": "^1.5.0",
-    "tslint": "3.15.1",
-    "tslint-microsoft-contrib": "5.0.3",
-    "typescript": "2.1.4"
+    "powerbi-models": "^1.0.3",
```

Comment:
Please add required dependencies into `dependencies`. Others must be kept in `devDependencies`